test/Bridges/Variable/zeros.jl

```diff
@@ -84,14 +84,20 @@ function test_zeros()
     )
     @test_throws err MOI.delete(bridged_mock, cyz)
 
-    err = ErrorException(
+    err = MOI.Bridges.Variable.CannotUnbridgeVariableFunction(
         "Cannot unbridge function because some variables are bridged by" *
         " variable bridges that do not support reverse mapping, for example," *
         " `ZerosBridge`.",
     )
     @test_throws err MOI.get(bridged_mock, MOI.ObjectiveFunction{typeof(obj)}())
     # With `c1`, the function does not contain any variable so it tests that it
-    # also throws an error even if it never calls `variable_unbridged_function`.
+    # also throws an error even if it never calls `variable_unbridged_function`
+    err = MOI.GetAttributeNotAllowed(
+        MOI.ConstraintFunction(),
+        "Cannot unbridge function because some variables are bridged by" *
+        " variable bridges that do not support reverse mapping, for example," *
+        " `ZerosBridge`.",
+    )
     @test_throws err MOI.get(bridged_mock, MOI.ConstraintFunction(), c1)
```

Comment:
@bleat here's an example. For `ConstraintFunction`, we throw `GetAttributeNotAllowed`. But for `ObjectiveFunction`, we throw `CannotUnbridgeVariableFunction`. What do you think should happen in each case?

Reply:
Bump @blegat

Reply:
We could add a `try`-`catch` here as well: https://github.com/jump-dev/MathOptInterface.jl/blob/b64dbe150ce54282718dd8d70d7a52cdfed5965f/src/Bridges/bridge_optimizer.jl#L1128
and throw `GetAttributeNotAllowed`